### PR TITLE
Allow passing messages

### PR DESCRIPTION
**Pull request by tlgimenes**
After the new feature of importing messages across apps, we need to be able to define the `messages` object to be passed to `IntProvider` since messages may not be defined in the app anymore. 

According to the typings of the `render` method, this passing the `messages` object was supported. However, the js code was not supporting this anymore. This PR makes this feature to work again

### CODE REVIEW

File: react.js [hunk 1]
```diff
@@ -31,15 +31,10 @@ const generateCacheKey = value => {
...
-    : {}
-
-  const intlProps = {
-    locale: locale || 'en',
-    messages: messages,
-  }
+  const locale = getLocale(path(['locale'], options)) || 'en'
+  const messages = options.messages || require(paths.resolveAppPath(`../messages/${locale}.json`))
+
+  const intlProps = { locale, messages, textComponent: 'span' }
```
Comment (lucasecdb) [inline react.js]: shouldn't we use a fragment for the text component? so it's more aligned with what render-runtime does

Comment (tlgimenes) [reply react.js]: I think so. I've used span because our snapshots are using span.

**Comment by tlgimenes** [comment]
Yes, the developer needs to do this:
```
const result = render(<Foobar />, { messages: { bar: 'Hello' } }>
```

This is because this lib is run in the dev's machine. Once we migrate this to builder-hub we will be able to infer these messages automatically and provide a much better dev XP.

File: react.js (hunk 1 above)
Comment (tlgimenes) [reply react.js]: I've just changed to fragment. Keep in mind some snapshots will break though

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [3.1.0-beta.4] - 2020-02-17
+
+## [3.1.0-beta.3] - 2020-02-16
+
+## [3.1.0-beta.2] - 2020-02-16
```
Comment (salesfelipe) [inline CHANGELOG.md]: Beta releases shouldn't be made on the branch that will be merged into master...

**Comment by tlgimenes** [comment]
After talking to @lbebber , we've decided to use the id as fallback in storecomponents tests

**Comment by tlgimenes** [comment]
After talking to @lbebber , we've decided to use the id as fallback in the tests, thus not needing to do this update

**Pull request closed without merging**